10316.xml

```diff
@@ -0,0 +1,233 @@
+    <Name>RCU</Name>
+    <Description1><![CDATA[This LWM2M Object includes RCU(Robotic Control Unit) information and the link to RCU APPs.]]></Description1>
+    <ObjectID>10316</ObjectID>
+    <ObjectURN>urn:oma:lwm2m:x:10316</ObjectURN>
+    <LWM2MVersion>1.0</LWM2MVersion>
+    <ObjectVersion>1.0</ObjectVersion>
+    <MultipleInstances>Single</MultipleInstances>
+    <Mandatory>Mandatory</Mandatory>
+    <Resources>
+      <Item ID="1">
+        <Name>RCU ID</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>String</Type>
+        <RangeEnumeration>0..127</RangeEnumeration>
+        <Units/>
+        <Description><![CDATA[The RCU identity.]]></Description>
+      </Item>
+      <Item ID="2">
+        <Name>RCU Serial Number</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>String</Type>
+        <RangeEnumeration>0..63</RangeEnumeration>
+        <Units/>
+        <Description><![CDATA[Serial Number of the RCU.]]></Description>
+      </Item>
+      <Item ID="3">
+        <Name>RCU Software Version</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>String</Type>
+        <RangeEnumeration>0..63</RangeEnumeration>
+        <Units/>
+        <Description><![CDATA[Software Version of the RCU.]]></Description>
+      </Item>
+      <Item ID="4">
+        <Name>RCU OS Version</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>String</Type>
+        <RangeEnumeration>0..127</RangeEnumeration>
+        <Units/>
+        <Description><![CDATA[OS type and version of the RCU.]]></Description>
+      </Item>	  
+	  <Item ID="5">
+        <Name>RCU CPU Info</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>String</Type>
+        <RangeEnumeration>64</RangeEnumeration>
+        <Units/>
+        <Description><![CDATA[Cpu information of the RCU.]]></Description>
+      </Item>
+	  <Item ID="6">
+        <Name>RCU RAM Info</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>String</Type>
+        <RangeEnumeration>64</RangeEnumeration>
+        <Units/>
+        <Description><![CDATA[RAM information of the RCU.]]></Description>
+      </Item>
+	  <Item ID="7">
+        <Name>RCU ROM Size</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration/>
+        <Units>G</Units>
```

Comment:
Incorrect Unit submitted - The Units defined in OMA LwM2M Objects are defined by SenML - Sensor Measurement Lists.

Reply:
Should be GB, the unit of storage, gigabyte.

Reply:
Gb should not be used as they are commonly used to represent other measurements.
See https://tools.ietf.org/html/rfc8428

Reply:
Changed GB to gigaB